packages/react-native-lockdown/src/repair.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Repair JS intrinsics
+ *
+ * The Hermes (or vanilla) SES shim is loaded as the first Metro polyfill, so
+ * {@link repairIntrinsics} can be called.
+ *
+ * Called with lockdown options ideal for React Native apps.
+ *
+ * {@link lockdown} itself is not called to allow for vetted shims as Metro
+ * polyfills before hardening (e.g. RN JS polyfills).
+ */
+
+// NB: ES5
+
+// eslint-disable-next-line no-var
+var dumbPolyfillExposedInternals = Object.entries(Promise).filter(
```

Comment:
if arrow callbacks are disallowed in this workspace, please create an `.eslintrc` and disable it

Reply:
rewrote var to const and removed both stale eslint disables

arrow fn is only not allowed in the file, i've clarified above in packageDocumentation